api/src/utils/sites.ts

```diff
@@ -0,0 +1,20 @@
+import { users } from "../constants/users"
+
+export const SITE_TO_SHOP_IDS: Record<string, number> = {
+  pug: 1,
+  stitch: 2,
+  luminara: 3,
+}
+
+export const getShopIdBySite = (site: string): number | undefined =>
+  SITE_TO_SHOP_IDS[site]
+
+export const getUserBySite = (site: string) => {
```

Comment:
This is needed for the `/metabase/sso` endpoint, so we know which user to use for signing the JWT, to send to the Metabase instance.